src/components/catalog-filter/catalog-filter.html

```diff
@@ -0,0 +1,22 @@
+<pf-filter-panel config="$ctrl.config">
+  <div class="filter-panel-container">
+    <input type="text" ng-model="$ctrl.keywordFilter.value"
+           class="keyword-filter"
+           placeholder="{{$ctrl.keywordFilter.placeholder}}"
+           ng-keypress="$ctrl.onKeywordKeyPress($event)"
+           ng-disabled="$ctrl.config.totalCount <= 1"/>
+    <div class="category" ng-repeat="filter in $ctrl.filterPanelModel" ng-if="!$first">
+      {{filter.title}}
+      <span class="pficon pficon-warning-triangle-o vendor-warning"
+            data-toggle="tooltip" aria-hidden="true" data-original-title="This filter will only apply to Catalog Items which contain Vendor information.
+            Catalog Items which do not have Vendor information will not be shown in the filter results.">
+      </span>
+      <ul>
+        <li ng-repeat="value in filter.values">
+          <input type="checkbox" ng-model="value.selected" ng-change="$ctrl.filterChanged()"/>
+          <span class="category-option-label">{{value.title}}</span>
```

Comment:
Put a label element around the input so screen readers know what the label value is for the checkbox.

```html
<div class="checkbox">
  <label>
    <input type="checkbox" ng-model="value.selected" ng-change="$ctrl.filterChanged()">
    <span class="category-option-label">{{value.title}}</span>
  </label>
</div>
```

(If the checkbox div causes layout issues, that can be removed.)